model/installation.go

```diff
@@ -35,15 +35,14 @@ type Installation struct {
 	Size                       string
 	Affinity                   string
 	State                      string
+	CRVersion                  string
```

Comment:
Did you mean to drop the comment & JSON annotation?

Reply:
Yep, the migration should have not left any installations with an empty value so best not to hide this in case we get a bug one day. Thanks for checking though!